addons/payment/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,101 @@
+---Fields in module 'payment'---
+payment      / payment.acquirer         / auto_confirm (selection)      : NEW required: required, selection_keys: ['at_pay_confirm', 'at_pay_now', 'none'], req_default: at_pay_confirm
+# Nothing to do
+
+payment      / payment.acquirer         / cancel_msg (html)             : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / done_msg (html)               : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / error_msg (html)              : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / image (binary)                : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / pending_msg (html)            : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / registration_view_template_id (many2one): NEW relation: ir.ui.view
+# Nothing to do
+
+payment      / payment.acquirer         / sequence (integer)            : NEW
+# Nothing to do
+
+payment      / payment.acquirer         / validation (selection)        : DEL selection_keys: ['automatic', 'manual']
```

Comment:
If the column is renamed, where is the line with the `NEW`entry? You should join both lines here in the work file.